gapic/schema/metadata.py

```diff
@@ -188,6 +192,28 @@ def python_import(self) -> imp.Import:
                 alias=self.module_alias,
             )
 
+        if self.is_proto_plus_type:
+            # We need to change the import statement to use an
+            # underscore between the module and the version. For example,
+            # change google.cloud.documentai.v1 to google.cloud.documentai_v1.
+            # Check if the package name contains a version.
+            version_regex = "^v\d(alpha|beta)?\d?"
```

Comment:
```suggestion
            version_regex = "^v\d+(alpha|beta)?\d*"
```

Reply:
We could have double digit versions on either side of the preview labels.

Also, we have seen some more complicated version numbers like [these](https://github.com/googleapis/googleapis/tree/master/google/cloud/asset) (v1p5beta1). So maybe the regex should be `"^v\d[^/]*$"` ? WDYT?

Reply:
Fixed